ui/src/pages/frontsite/components/ILSMenu/ILSMenu.js

```diff
@@ -18,27 +25,40 @@ export default class ILSMenu extends Component {
   };
 
   renderRightDropDown = userMenuText => {
+    const dropdownEntries = (
+      <Dropdown.Menu>
+        <Dropdown.Item as={Link} to={FrontSiteRoutes.patronProfile}>
+          Your Profile
```

Comment:
```suggestion
          Your loans
```
We will have a conflict anyway